Library/Homebrew/vendor/bundle/bundler/setup.rb

```diff
@@ -13,10 +13,10 @@
 $:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/gems/ast-2.4.1/lib"
 $:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/gems/bindata-2.4.8/lib"
 $:.unshift "#{path}/"
-$:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/extensions/universal-darwin-19/2.6.0/byebug-11.1.3"
+$:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/extensions/x86_64-linux/2.6.0-static/byebug-11.1.3"
```

Comment:
This chunk doesn't look right to me. What's going on here?

Reply:
Yeah, this probably needs to be reverted.